docs/contributing.rst

```diff
@@ -97,7 +97,7 @@ the repository, you can do something like the following::
 
 To verify that your development environment is working, you can run the unit tests::
 
-    $ pytest -v zarr
+    $ python -m pytest -v zarr
```

Comment:
There is another line like this further down. Should we update that as well?

Any other command line calls like this we should update?

Reply:
@raybellwaves ?